tools/istio-docker.mk

```diff
@@ -211,30 +226,25 @@ $(foreach TGT,$(DOCKER_TARGETS),$(eval tar.$(TGT): $(TGT) | $(ISTIO_DOCKER_TAR)
    time (docker save -o ${ISTIO_DOCKER_TAR}/$(subst docker.,,$(TGT)).tar $(HUB)/$(subst docker.,,$(TGT)):$(TAG) && \
          gzip ${ISTIO_DOCKER_TAR}/$(subst docker.,,$(TGT)).tar)))
 
-# create a DOCKER_TAR_TARGETS that's each of DOCKER_TARGETS with a tar. prefix
-DOCKER_TAR_TARGETS:=
```

Comment:
may need to add this back in for the tar targets.

Reply:
what tar targers? WE are not creating tarballs of docker images. Only bazel needed it

Reply:
@rshriram T&R indicated they use tar (docker save) targets.  Not sure why.  I did test this and it works as is, but I'm going to revert the DOCKER_TAR_TARGETS:= because it can permit an environmental variable override (dirty environment) mucking things up.

Reply:
i just realized, you may have asked where the tar targets are added (down the page a bit - possibly not shown in the review via a add operator.